### ISSUE TITLE
Python error (solved)

### ISSUE THREAD

**Issue by elbogi** [software-2/nanoleaf#6]
..

**Comment by StudioHawk** [comment]
What was the solution here? I'm having a problem getting the pip install command to work and getting an error

**Comment by elbogi** [comment]
If I remember correctly, I ran the install with pip3 instead of pip. Then I ran the commands in a python3 shell.

**Comment by StudioHawk** [comment]
Legend! Thank you so much. If I'm correct, you were also using hass.io correct? How did you create the custo_components folder?

**Comment by elbogi** [comment]
The custom_components is a folder created inside the config-folder of your hassio installation. If, not just create it yourself, either with the HASS configurator or Samba addon in hassio.

**Comment by StudioHawk** [comment]
Thanks! Yeah just had to create it, think I'm stuck on the same point of you where it is saying that it isn't loading the component.

**Comment by elbogi** [comment]
Ok, so I ditched running HA in an virtual env (as this was just to test HA). Installed an Ubuntu 16.04 server and installed home assistant as described [here](https://home-assistant.io/docs/installation/python/). Then I copied the aurora.py to the `custom_components/light` folder. Added the old config to configuration.yaml, restarted HA and got the same module-error. I then installed nanoleaf manually, but I had to install with pip3 instead, so basically I just ran: `sudo pip3 install nanoleaf --upgrade`. Restartet HA and my Aurora Nanoleaf came to life! :-D
